ocaml/parsing/jane_syntax.mli

```diff
@@ -148,6 +148,22 @@ module Layouts : sig
   val type_of :
     loc:Location.t -> attrs:Parsetree.attributes ->
     core_type -> Parsetree.core_type
+
+  type nonrec extension_constructor =
+    (* [ 'a ('b : immediate) ('c : float64). 'a * 'b * 'c -> exception ] *)
+    | Lext_decl of (string Location.loc *
+                    Asttypes.layout_annotation option) list *
+                   Parsetree.constructor_arguments *
+                   Parsetree.core_type option
```

Comment:
Once again, I think a comment about when this shows up would be good -- is it for every extension constructor?  Just when there are quantified variables?  Just when those quantified variables have layouts?